### DISCUSSION TITLE
zhop after that first extrusion line and brim lines being far away from each other, please help

### DISCUSSION THREAD

**Discussion by coldasicee** [supermerill/SuperSlicer#2272] [Q&A]
Hi guys, how are you doing? I'm new to SuperSlicer (actually new to 3d printing, I have mine for 2 weeks only. I have 2 questions comparing to Cura and I was wondering if I could get that configured on my superslicer since I loved this software.

1. Cura usually does a Zhop after printing that initial extrusion line to clean that filament that gets released during heating, how can I configure that on SS?
2. During the calibrations offered by the software (big cudos for that!) I noticed that the brim have a considerate amount of space between each round, how can I get them to stick together?

NOTE: My bed leveling is OK.

- Extra: Also, I got an Ender 3 V2, am I the only one who things it's "super improved bed, with super tech texture" almost never sticks the first layer? Is there a fix for that (besides glue stick)? I even downloaded a mesh leveling firmware to try and fix it but it didn't help at all.

Thanks guys, sorry for the newbie questions and happy printing!
Here is a puppy to make your day maybe a little brighter lol.
![puppy-nawpic-2](https://user-images.githubusercontent.com/84877944/151098031-64d4f76b-19da-49ea-be35-ef0887520480.jpg)

**Comment by coldasicee** [comment]
Update, actually it isn't just the brim, the infill is having the same problem, image attached.
![20220126_022843](https://user-images.githubusercontent.com/84877944/151109091-041a17ce-9882-4720-92a2-8469e840fda0.jpg)

**Reply by supermerill** [reply]
> During the calibrations offered by the software (big cudos for that!) I noticed that the brim have a considerate amount of space between each round, how can I get them to stick together?

Are you sure your bed is levelled? and the flow good?

**Reply by coldasicee** [reply]
> Are you sure your bed is levelled? and the flow good?

As far as I know leveling was good, flow I did the calibration, I will check the bed again because I'm still using those sh**ty springs that comes with the machine (I started 2 weeks ago)

> you can use the min height for travel if you need it.
> Or you can do it yourself in your start gcode, after your purge line command.

I'll take a look at that gcode dictionary and see if I can do that, mind if I share with you if I manage?

>i have the first ender3, I have added a magnetic sticker and I use a steel plate, on which I stick a new pei on a regular basis, as it's easily ruined.

oof, I'll just searched for it and it's kinda expensive in my country, how long does it lasts regularly? It's around 50USD here converted lol.

**Comment by supermerill** [comment]
1. you can use the `min height for travel` if you need it.
Or you can do it yourself in your start gcode, after your purge line command.

**Reply by supermerill** [reply]
> I'll take a look at that gcode dictionary and see if I can do that, mind if I share with you if I manage?

you said you have a purge line, so something like `G1 X42.42 Y42.42 E4.242` to move Z up, just put a `G1 Z1` to move it to 1mm height

**Comment by supermerill** [comment]
> Extra: Also, I got an Ender 3 V2, am I the only one who things it's "super improved bed, with super tech texture" almost never sticks the first layer? Is there a fix for that (besides glue stick)? I even downloaded a mesh leveling firmware to try and fix it but it didn't help at all.

i have the first ender3, I have added a magnetic sticker and I use a steel plate, on which I stick a new pei on a regular basis, as it's easily ruined.

**Reply by supermerill** [reply]
> oof, I'll just searched for it and it's kinda expensive in my country, how long does it lasts regularly? It's around 50USD here converted lol.

3 for 10 bucks on amazon:
https://www.amazon.com/s?k=ender+sticker+sheet&crid=S7EADCN5KUMQ&sprefix=ender+sticker+sheet%2Caps%2C136&ref=nb_sb_noss_2
https://www.amazon.com/Bilway-Platform-Sticker-235x235mm-Creality/dp/B09BDZ6G36/ref=sr_1_7?crid=S7EADCN5KUMQ&keywords=ender+sticker+sheet&qid=1643231103&sprefix=ender+sticker+sheet%2Caps%2C136&sr=8-7

1 for 3.3 bucks in aliexpress:
https://fr.aliexpress.com/wholesale?catId=0&initiative_id=SB_20220126131430&isPremium=y&SearchText=ender+sticker+sheet
https://fr.aliexpress.com/item/32891707309.html?spm=a2g0o.productlist.0.0.2ce476bfFs82R5&algo_pvid=c9cf0c3b-95a6-4509-8af4-a92fe88d36a8&algo_exp_id=c9cf0c3b-95a6-4509-8af4-a92fe88d36a8-3&pdp_ext_f=%7B%22sku_id%22%3A%2265708357247%22%7D&pdp_pi=-1%3B3.23%3B-1%3B-1%40salePrice%3BEUR%3Bsearch-mainSearch
